diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 7e27e88..49a5073 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -42,12 +42,12 @@ class ChurroTimeEntryStatsHelper
                 continue;
             }
 
-            if (isset($types[$timeEntry->getType()])) {
-                $types[$timeEntry->getType()][] = $timeEntry->getQuantityMade();
-            } else {
+            // initialize key if necessary
+            if (!isset($types[$timeEntry->getType()])) {
                 $types[$timeEntry->getType()] = [];
-                $types[$timeEntry->getType()][] = $timeEntry->getQuantityMade();
             }
+
+            $types[$timeEntry->getType()][] = $timeEntry->getQuantityMade();
         }
 
         $bestType = null;
