diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 1bbc1a0..218c986 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -80,10 +80,14 @@ class ChurroTimeEntryStatsHelper
             return false;
         }
 
-        if (($today->format('j') === 30 || $today->format('j') === 31)
-            && $today->format('n') !== 10) {
+        // always use the filter in October
+        if ($today->format('n') === 10) {
+            return false;
+        }
+
+        if (($today->format('j') === 30 || $today->format('j') === 31)) {
             // don't use filter if today is 30th/31st of July-December
-            // except for October - always use the filter in October
+
             $useFilter = false;
         }
 
