diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 81df331..8281ea8 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -72,7 +72,6 @@ class ChurroTimeEntryStatsHelper
 
     private function shouldUseTimeFilter()
     {
-        $useFilter = true;
         $today = new \DateTime('now');
 
         if ($today->format('n') <= 6) {
@@ -87,10 +86,9 @@ class ChurroTimeEntryStatsHelper
 
         if (($today->format('j') === 30 || $today->format('j') === 31)) {
             // don't use filter if today is 30th/31st of July-December
-
-            $useFilter = false;
+            return false;
         }
 
-        return $useFilter;
+        return true;
     }
 }
