diff --git a/src/AppBundle/DataFixtures/ORM/DummyDataFixtures.php b/src/AppBundle/DataFixtures/ORM/DummyDataFixtures.php
index e3b52bb..b13f534 100644
--- a/src/AppBundle/DataFixtures/ORM/DummyDataFixtures.php
+++ b/src/AppBundle/DataFixtures/ORM/DummyDataFixtures.php
@@ -18,13 +18,7 @@ class DummyDataFixtures implements FixtureInterface
     {
         $this->createBakers($manager);
 
-        $types = [
-            'plain',
-            'guava',
-            'chocolate',
-            'dulce de leche',
-            'vanilla'
-        ];
+        $types = ChurroTimeEntry::VALID_CHURRO_TYPES;
 
         for ($i = 0; $i < 20; $i++) {
             $startCookingTime = \DateTime::createFromFormat(
diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 8dcd0a1..96dee58 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -12,6 +12,14 @@ use Symfony\Component\Validator\Context\ExecutionContextInterface;
  */
 class ChurroTimeEntry
 {
+    const VALID_CHURRO_TYPES = [
+        'plain',
+        'guava',
+        'chocolate',
+        'dulce de leche',
+        'vanilla'
+    ];
+
     /**
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="AUTO")
