diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index da0684f..2ec00e6 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -26,7 +26,7 @@ class ChurroTimeEntryController extends Controller
 
         $statsHelper = $this->container
             ->get('app.churro_time_entry_stats_helper');
-        $bestTypeData = $statsHelper->getMostEfficientTypeData($timeEntries);
+        $bestTypeData = $statsHelper->getMostEfficientTypeData();
 
         return $this->render('AppBundle:ChurroTimeEntry:list.html.twig', [
             'timeEntries' => $timeEntries,
diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 380e6d0..854a8c9 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -10,10 +10,9 @@ class ChurroTimeEntryStatsHelper
      * Determines which "type" cooks the most churros on average
      * and returns that inside an array with "type" and "average" keys.
      *
-     * @param ChurroTimeEntry[] $timeEntries
      * @return array
      */
-    public function getMostEfficientTypeData(array $timeEntries)
+    public function getMostEfficientTypeData()
     {
         $timeEntries = $this->container->get('doctrine')
             ->getRepository(ChurroTimeEntry::class)
