diff --git a/src/AppBundle/Controller/DefaultController.php b/src/AppBundle/Controller/DefaultController.php
index 45fdb65..c6118a8 100644
--- a/src/AppBundle/Controller/DefaultController.php
+++ b/src/AppBundle/Controller/DefaultController.php
@@ -10,8 +10,13 @@ class DefaultController extends Controller
 {
     public function homepageAction()
     {
-        return $this->render('AppBundle:Default:homepage.html.twig', array(
+        $statsHelper = $this->container
+            ->get('app.churro_time_entry_stats_helper');
+        $bestTypeData = $statsHelper->getMostEfficientTypeData();
 
+        return $this->render('AppBundle:Default:homepage.html.twig', array(
+            'bestType' => $bestTypeData['type'],
+            'avg' => $bestTypeData['average'],
         ));
     }
 }
diff --git a/src/AppBundle/Resources/views/Default/homepage.html.twig b/src/AppBundle/Resources/views/Default/homepage.html.twig
index 05f26be..e309a23 100644
--- a/src/AppBundle/Resources/views/Default/homepage.html.twig
+++ b/src/AppBundle/Resources/views/Default/homepage.html.twig
@@ -3,5 +3,13 @@
 {% block body %}
     <main class="container">
         <h1>Bienvenidos a la Churreria!</h1>
+
+        <dl class="row" style="border: 2px solid #f7f7f9;">
+            <dt class="col-sm-8">Most efficient type</dt>
+            <dd class="col-sm-4">{{ bestType }}</dd>
+
+            <dt class="col-sm-8">Average Made for type</dt>
+            <dd class="col-sm-4">{{ avg }}</dd>
+        </dl>
     </main>
 {% endblock %}
