diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 317b66d..003e5c0 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -19,6 +19,10 @@ class ChurroTimeEntryController extends Controller
             ->getQuery()
             ->getResult();
 
+        $this->container
+            ->get('logger')
+            ->info(sprintf('Printing %d time entries', count($timeEntries)));
+
         $bestTypeData = $this->getMostEfficientTypeData($timeEntries);
 
         return $this->render('AppBundle:ChurroTimeEntry:list.html.twig', [
