diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 34bd15b..8bd6468 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -34,6 +34,11 @@ class ChurroTimeEntryStatsHelper
 
         $types = [];
         foreach ($timeEntries as $timeEntry) {
+            if ($timeEntry->getBakedBy()->getUsername() === 'jwage') {
+                // jwage doesn't count
+                continue;
+            }
+
             if ($useFilter && $timeEntry->getStartCookingAt()->format('H') < 6) {
                 continue;
             }
