diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index c8a1322..e0117fb 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -3,12 +3,13 @@
 namespace AppBundle\Service;
 
 use AppBundle\Entity\ChurroTimeEntry;
+use Doctrine\Bundle\DoctrineBundle\Registry;
 
 class ChurroTimeEntryStatsHelper
 {
     private $doctrine;
 
-    public function __construct($doctrine)
+    public function __construct(Registry $doctrine)
     {
         $this->doctrine = $doctrine;
     }
