diff --git a/src/AppBundle/Form/ChurroTimeEntryForm.php b/src/AppBundle/Form/ChurroTimeEntryForm.php
index 701bb9c..89bbf85 100644
--- a/src/AppBundle/Form/ChurroTimeEntryForm.php
+++ b/src/AppBundle/Form/ChurroTimeEntryForm.php
@@ -9,5 +9,13 @@ class ChurroTimeEntryForm extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $builder
+            ->add('type')
+            ->add('startCookingAt')
+            ->add('endCookingAt')
+            ->add('startCleanupAt')
+            ->add('endCleanupAt')
+            ->add('quantityMade')
+        ;
     }
 }
