diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 4af9004..6348fdc 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -3,6 +3,7 @@
 namespace AppBundle\Controller;
 
 use AppBundle\Entity\ChurroTimeEntry;
+use AppBundle\Form\ChurroTimeEntryForm;
 use AppBundle\Service\ChurroTimeEntryStatsHelper;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
@@ -56,8 +57,11 @@ class ChurroTimeEntryController extends Controller
             throw $this->createNotFoundException('no time entry for '.$id);
         }
 
+        $form = $this->createForm(ChurroTimeEntryForm::class, $timeEntry);
+
         return $this->render('AppBundle:ChurroTimeEntry:edit.html.twig', [
-            'timeEntry' => $timeEntry
+            'timeEntry' => $timeEntry,
+            'form' => $form->createView(),
         ]);
     }
 }
diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/edit.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/edit.html.twig
index 91dce72..7b3225c 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/edit.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/edit.html.twig
@@ -3,5 +3,11 @@
 {% block body %}
     <div class="container">
         <h1>Editing Entry {{ timeEntry.id }}</h1>
+
+        {{ form_start(form) }}
+            {{ form_widget(form) }}
+
+            <button type="submit" class="btn btn-primary">Save</button>
+        {{ form_end(form) }}
     </div>
 {% endblock %}
