diff --git a/src/AppBundle/Form/ChurroTimeEntryForm.php b/src/AppBundle/Form/ChurroTimeEntryForm.php
index 84eca1a..fd96098 100644
--- a/src/AppBundle/Form/ChurroTimeEntryForm.php
+++ b/src/AppBundle/Form/ChurroTimeEntryForm.php
@@ -3,6 +3,7 @@
 namespace AppBundle\Form;
 
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\FormBuilderInterface;
@@ -12,7 +13,12 @@ class ChurroTimeEntryForm extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('type')
+            ->add('type', ChoiceType::class, [
+                'choices' => [
+                    'vanilla' => 'Vanilla',
+                    'chocolate' => 'Chocolate',
+                ]
+            ])
             ->add('startCookingAt', DateTimeType::class, [
                 'widget' => 'single_text'
             ])
