diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 4135c19..d5bc25d 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -6,6 +6,7 @@ use AppBundle\Entity\ChurroTimeEntry;
 use AppBundle\Form\ChurroTimeEntryForm;
 use AppBundle\Service\ChurroTimeEntryStatsHelper;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 class ChurroTimeEntryController extends Controller
@@ -46,11 +47,16 @@ class ChurroTimeEntryController extends Controller
         ]);
     }
 
-    public function newAction()
+    public function newAction(Request $request)
     {
         $form = $this->createForm(ChurroTimeEntryForm::class, null, [
             'csrf_protection' => false,
         ]);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            var_dump($form->getData());die;
+        }
 
         return $this->render('AppBundle:ChurroTimeEntry:new.html.twig', [
             'form' => $form->createView(),
