diff --git a/src/AppBundle/Form/ChurroTimeEntryForm.php b/src/AppBundle/Form/ChurroTimeEntryForm.php
index fd96098..89783e2 100644
--- a/src/AppBundle/Form/ChurroTimeEntryForm.php
+++ b/src/AppBundle/Form/ChurroTimeEntryForm.php
@@ -17,7 +17,8 @@ class ChurroTimeEntryForm extends AbstractType
                 'choices' => [
                     'vanilla' => 'Vanilla',
                     'chocolate' => 'Chocolate',
-                ]
+                ],
+                'placeholder' => 'Choose a type'
             ])
             ->add('startCookingAt', DateTimeType::class, [
                 'widget' => 'single_text'
