diff --git a/src/AppBundle/Model/ChurroTypeStats.php b/src/AppBundle/Model/ChurroTypeStats.php
index 0cd0bec..8a91dcd 100644
--- a/src/AppBundle/Model/ChurroTypeStats.php
+++ b/src/AppBundle/Model/ChurroTypeStats.php
@@ -8,23 +8,19 @@ class ChurroTypeStats
 
     private $averageQuantityMade;
 
-    public function getType()
+    public function __construct($type, $averageQuantityMade)
     {
-        return $this->type;
+        $this->type = $type;
+        $this->averageQuantityMade = $averageQuantityMade;
     }
 
-    public function setType($type)
+    public function getType()
     {
-        $this->type = $type;
+        return $this->type;
     }
 
     public function getAverageQuantityMade()
     {
         return $this->averageQuantityMade;
     }
-
-    public function setAverageQuantityMade($averageQuantityMade)
-    {
-        $this->averageQuantityMade = $averageQuantityMade;
-    }
 }
diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 4391250..e6f5421 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -79,9 +79,7 @@ class ChurroTimeEntryStatsHelper
 
         $this->logger->info('Most efficient type is '.$bestType);
 
-        $typeStats=  new ChurroTypeStats();
-        $typeStats->setType($bestType);
-        $typeStats->setAverageQuantityMade($avg);
+        $typeStats = new ChurroTypeStats($bestType, $avg);
 
         return $typeStats;
     }
