diff --git a/src/AppBundle/Model/ChurroTypeStats.php b/src/AppBundle/Model/ChurroTypeStats.php
index ca10f33..0cd0bec 100644
--- a/src/AppBundle/Model/ChurroTypeStats.php
+++ b/src/AppBundle/Model/ChurroTypeStats.php
@@ -6,7 +6,7 @@ class ChurroTypeStats
 {
     private $type;
 
-    private $average;
+    private $averageQuantityMade;
 
     public function getType()
     {
@@ -18,13 +18,13 @@ class ChurroTypeStats
         $this->type = $type;
     }
 
-    public function getAverage()
+    public function getAverageQuantityMade()
     {
-        return $this->average;
+        return $this->averageQuantityMade;
     }
 
-    public function setAverage($average)
+    public function setAverageQuantityMade($averageQuantityMade)
     {
-        $this->average = $average;
+        $this->averageQuantityMade = $averageQuantityMade;
     }
 }
diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 94d60d3..62d7391 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -81,7 +81,7 @@ class ChurroTimeEntryStatsHelper
 
         $typeStats=  new ChurroTypeStats();
         $typeStats->setType($bestType);
-        $typeStats->setAverage($avg);
+        $typeStats->setAverageQuantityMade($avg);
 
         return $typeStats;
     }
