diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index d711fd4..ea445ff 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -25,8 +25,7 @@ class ChurroTimeEntryController extends Controller
 
         return $this->render('AppBundle:ChurroTimeEntry:list.html.twig', [
             'timeEntries' => $timeEntries,
-            'bestType' => $bestTypeData['type'],
-            'avg' => $bestTypeData['average'],
+            'bestTypeStats' => $bestTypeData,
         ]);
     }
 
diff --git a/src/AppBundle/Controller/DefaultController.php b/src/AppBundle/Controller/DefaultController.php
index c6118a8..0ac5e14 100644
--- a/src/AppBundle/Controller/DefaultController.php
+++ b/src/AppBundle/Controller/DefaultController.php
@@ -15,8 +15,7 @@ class DefaultController extends Controller
         $bestTypeData = $statsHelper->getMostEfficientTypeData();
 
         return $this->render('AppBundle:Default:homepage.html.twig', array(
-            'bestType' => $bestTypeData['type'],
-            'avg' => $bestTypeData['average'],
+            'bestTypeStats' => $bestTypeData,
         ));
     }
 }
diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/_bestChurroTypeBox.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/_bestChurroTypeBox.html.twig
index c487f4d..2f3d557 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/_bestChurroTypeBox.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/_bestChurroTypeBox.html.twig
@@ -1,7 +1,7 @@
 <dl class="row" style="border: 2px solid #f7f7f9;">
     <dt class="col-sm-8">Most efficient type</dt>
-    <dd class="col-sm-4">{{ bestType }}</dd>
+    <dd class="col-sm-4">{{ bestTypeStats.type }}</dd>
 
     <dt class="col-sm-8">Average Made for type</dt>
-    <dd class="col-sm-4">{{ avg }}</dd>
+    <dd class="col-sm-4">{{ bestTypeStats.averageQuantityMade }}</dd>
 </dl>
\ No newline at end of file
