diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 6c6188c..622804a 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -3,6 +3,7 @@
 namespace AppBundle\Controller;
 
 use AppBundle\Entity\ChurroTimeEntry;
+use AppBundle\Form\ChurroTimeEntryForm;
 use AppBundle\Service\ChurroTimeEntryStatsHelper;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
@@ -47,8 +48,10 @@ class ChurroTimeEntryController extends Controller
 
     public function newAction()
     {
-        return $this->render('AppBundle:ChurroTimeEntry:new.html.twig', [
+        $form = $this->createForm(ChurroTimeEntryForm::class);
 
+        return $this->render('AppBundle:ChurroTimeEntry:new.html.twig', [
+            'form' => $form->createView(),
         ]);
     }
 }
diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/new.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/new.html.twig
index d286b40..fbd8360 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/new.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/new.html.twig
@@ -3,5 +3,11 @@
 {% block body %}
     <div class="container">
         <h1>New Time Entry</h1>
+
+        {{ form_start(form) }}
+            {{ form_widget(form) }}
+
+            <button type="submit" class="btn btn-primary">Save</button>
+        {{ form_end(form) }}
     </div>
 {% endblock %}
