diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 1d2b49a..00ddc8a 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -44,6 +44,13 @@ class ChurroTimeEntryController extends Controller
         ]);
     }
 
+    /**
+     * Determines which "type" cooks the most churros on average
+     * and returns that inside an array with "type" and "average" keys.
+     *
+     * @param ChurroTimeEntry[] $timeEntries
+     * @return array
+     */
     private function getMostEfficientTypeData(array $timeEntries)
     {
         $types = [];
