diff --git a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
index 47a9435..c657f9e 100644
--- a/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
+++ b/src/AppBundle/Service/ChurroTimeEntryStatsHelper.php
@@ -27,12 +27,7 @@ class ChurroTimeEntryStatsHelper
     {
         $timeEntries = $this->doctrine
             ->getRepository(ChurroTimeEntry::class)
-            ->createQueryBuilder('churro_time_entry')
-            ->where('churro_time_entry.startCookingAt > :date')
-            ->setParameter('date', new \DateTime('-1 week'))
-            ->orderBy('churro_time_entry.startCookingAt', 'DESC')
-            ->getQuery()
-            ->getResult();
+            ->findAllDuringLastWeekOrderedNewestFirst();
 
         $useFilter = true;
         $today = new \DateTime('now');
