diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 55b4e39..ef885d0 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -5,7 +5,7 @@ namespace AppBundle\Entity;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @ORM\Entity()
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\ChurroTimeEntryRepository")
  * @ORM\Table()
  */
 class ChurroTimeEntry
diff --git a/src/AppBundle/Repository/ChurroTimeEntryRepository.php b/src/AppBundle/Repository/ChurroTimeEntryRepository.php
new file mode 100644
index 0000000..17865e6
--- /dev/null
+++ b/src/AppBundle/Repository/ChurroTimeEntryRepository.php
@@ -0,0 +1,10 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use Doctrine\ORM\EntityRepository;
+
+class ChurroTimeEntryRepository extends EntityRepository
+{
+
+}
