diff --git a/src/AppBundle/Repository/ChurroTimeEntryRepository.php b/src/AppBundle/Repository/ChurroTimeEntryRepository.php
index 17865e6..aeaf1d8 100644
--- a/src/AppBundle/Repository/ChurroTimeEntryRepository.php
+++ b/src/AppBundle/Repository/ChurroTimeEntryRepository.php
@@ -6,5 +6,15 @@ use Doctrine\ORM\EntityRepository;
 
 class ChurroTimeEntryRepository extends EntityRepository
 {
-
+    public function findAllDuringLastWeekOrderedNewestFirst()
+    {
+        $timeEntries = $this->doctrine
+            ->getRepository(ChurroTimeEntry::class)
+            ->createQueryBuilder('churro_time_entry')
+            ->where('churro_time_entry.startCookingAt > :date')
+            ->setParameter('date', new \DateTime('-1 week'))
+            ->orderBy('churro_time_entry.startCookingAt', 'DESC')
+            ->getQuery()
+            ->getResult();
+    }
 }
