diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index ecfe981..6a52fc6 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -75,6 +75,7 @@ class ChurroTimeEntryController extends Controller
 
     public function showAction($id)
     {
+        /** @var ChurroTimeEntry $timeEntry */
         $timeEntry = $this->getDoctrine()
             ->getRepository(ChurroTimeEntry::class)
             ->find($id);
