diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 07a8018..2e06a41 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -57,6 +57,7 @@ class ChurroTimeEntryController extends Controller
 
     public function showAction($id)
     {
+        /** @var ChurroTimeEntry $timeEntry */
         $timeEntry = $this->getDoctrine()
             ->getRepository(ChurroTimeEntry::class)
             ->find($id);
