diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 6794623..26f2562 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -75,6 +75,10 @@ class ChurroTimeEntryController extends Controller
 
     public function showAction($id)
     {
-        return new Response('Time entry '.$id);
+        $timeEntry = $this->getDoctrine()
+            ->getRepository(ChurroTimeEntry::class)
+            ->find($id);
+
+        return new Response('Time entry quantity '.$timeEntry->getQuantityMade());
     }
 }
