diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
index 53e2628..52f1266 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
@@ -25,6 +25,7 @@
                     <th>quantity made</th>
                     <th>cook time</th>
                     <th>cleanup time</th>
+                    <th>actions</th>
                 </tr>
             </thead>
             <tbody>
@@ -43,6 +44,13 @@
                         >
                             :{{ ((timeEntry.endCleanupAt.timestamp - timeEntry.startCleanupAt.timestamp) / 60)|round }}
                         </td>
+                        <td>
+                            <a href="{{ path('app_churro_time_entry_show', {
+                                id: timeEntry.id
+                            }) }}">
+                                view
+                            </a>
+                        </td>
                     </tr>
                 {% endfor %}
             </tbody>
