diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
index a9870fa..8e32a9a 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
@@ -3,6 +3,11 @@
 {% block body %}
     <div class="container">
         <h1>Time entry {{ timeEntry.id }}</h1>
+
+        <span class="badge badge-info">
+            baked by {{ timeEntry.bakedBy.firstName }} {{ timeEntry.bakedBy.lastName|slice(0, 1) }}.
+        </span>
+
         <table class="table">
             <tbody>
                 <tr>
