diff --git a/src/AppBundle/Entity/Baker.php b/src/AppBundle/Entity/Baker.php
index c9eeea8..2a95426 100644
--- a/src/AppBundle/Entity/Baker.php
+++ b/src/AppBundle/Entity/Baker.php
@@ -66,4 +66,9 @@ class Baker
     {
         $this->lastName = $lastName;
     }
+
+    public function getAbbreviatedName()
+    {
+        return $this->firstName . ' ' .substr($this->lastName, 0, 1) . '.';
+    }
 }
diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
index 8e32a9a..ce2d9ec 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/show.html.twig
@@ -5,7 +5,7 @@
         <h1>Time entry {{ timeEntry.id }}</h1>
 
         <span class="badge badge-info">
-            baked by {{ timeEntry.bakedBy.firstName }} {{ timeEntry.bakedBy.lastName|slice(0, 1) }}.
+            baked by {{ timeEntry.bakedBy.abbreviatedName }}
         </span>
 
         <table class="table">
