diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 7902f16..8c49eee 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -176,4 +176,12 @@ class ChurroTimeEntry
     {
         return $this->getEndCleanupAt()->getTimestamp() - $this->getStartCleanupAt()->getTimestamp();
     }
+
+    /**
+     * @return bool
+     */
+    public function didCleanupTakeTooLong()
+    {
+        return ($this->getCleanupDuration() / 60) > 30;
+    }
 }
diff --git a/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig b/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
index 86f8dcb..404a8ef 100644
--- a/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
+++ b/src/AppBundle/Resources/views/ChurroTimeEntry/list.html.twig
@@ -32,7 +32,7 @@
                             :{{ ((timeEntry.cookingDuration) / 60)|round }}
                         </td>
                         <td
-                            {% if ((timeEntry.cleanupDuration) / 60) > 30 %}
+                            {% if timeEntry.didCleanupTakeTooLong %}
                                 class="text-danger"
                             {% endif %}
                         >
