diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 953154a..6211287 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -76,7 +76,7 @@ class ChurroTimeEntry
         return $this->startCookingAt;
     }
 
-    public function setStartCookingAt($startCookingAt)
+    public function setStartCookingAt(\DateTime $startCookingAt)
     {
         $this->startCookingAt = $startCookingAt;
     }
@@ -89,7 +89,7 @@ class ChurroTimeEntry
         return $this->endCookingAt;
     }
 
-    public function setEndCookingAt($endCookingAt)
+    public function setEndCookingAt(\DateTime $endCookingAt)
     {
         $this->endCookingAt = $endCookingAt;
     }
@@ -102,7 +102,7 @@ class ChurroTimeEntry
         return $this->startCleanupAt;
     }
 
-    public function setStartCleanupAt($startCleanupAt)
+    public function setStartCleanupAt(\DateTime $startCleanupAt)
     {
         $this->startCleanupAt = $startCleanupAt;
     }
@@ -115,7 +115,7 @@ class ChurroTimeEntry
         return $this->endCleanupAt;
     }
 
-    public function setEndCleanupAt($endCleanupAt)
+    public function setEndCleanupAt(\DateTime $endCleanupAt)
     {
         $this->endCleanupAt = $endCleanupAt;
     }
