diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index ef885d0..9eca7b0 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -62,6 +62,9 @@ class ChurroTimeEntry
         $this->type = $type;
     }
 
+    /**
+     * @return \DateTime
+     */
     public function getStartCookingAt()
     {
         return $this->startCookingAt;
@@ -72,6 +75,9 @@ class ChurroTimeEntry
         $this->startCookingAt = $startCookingAt;
     }
 
+    /**
+     * @return \DateTime
+     */
     public function getEndCookingAt()
     {
         return $this->endCookingAt;
@@ -82,6 +88,9 @@ class ChurroTimeEntry
         $this->endCookingAt = $endCookingAt;
     }
 
+    /**
+     * @return \DateTime
+     */
     public function getStartCleanupAt()
     {
         return $this->startCleanupAt;
@@ -92,6 +101,9 @@ class ChurroTimeEntry
         $this->startCleanupAt = $startCleanupAt;
     }
 
+    /**
+     * @return \DateTime
+     */
     public function getEndCleanupAt()
     {
         return $this->endCleanupAt;
