diff --git a/src/AppBundle/Tests/Entity/BakerTest.php b/src/AppBundle/Tests/Entity/BakerTest.php
new file mode 100644
index 0000000..afebbe0
--- /dev/null
+++ b/src/AppBundle/Tests/Entity/BakerTest.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace AppBundle\Tests\Entity;
+
+use PHPUnit\Framework\TestCase;
+
+class BakerTest extends TestCase
+{
+    public function testGetAbbreviatedName()
+    {
+        $this->assertTrue(true);
+    }
+}
+
