diff --git a/src/AppBundle/Tests/Entity/BakerTest.php b/src/AppBundle/Tests/Entity/BakerTest.php
index b21e3f5..8c8b161 100644
--- a/src/AppBundle/Tests/Entity/BakerTest.php
+++ b/src/AppBundle/Tests/Entity/BakerTest.php
@@ -14,6 +14,11 @@ class BakerTest extends TestCase
         $baker->setLastName('Weaver');
 
         $this->assertSame('Ryan W.', $baker->getAbbreviatedName());
+
+        $baker = new Baker();
+        $baker->setFirstName('Ryan');
+
+        $this->assertSame('Ryan', $baker->getAbbreviatedName());
     }
 }
 
