diff --git a/src/AppBundle/Tests/Service/ChurroTimeEntryStatsHelperTest.php b/src/AppBundle/Tests/Service/ChurroTimeEntryStatsHelperTest.php
new file mode 100644
index 0000000..a84ea54
--- /dev/null
+++ b/src/AppBundle/Tests/Service/ChurroTimeEntryStatsHelperTest.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle\Tests\Service;
+
+use PHPUnit\Framework\TestCase;
+
+class ChurroTimeEntryStatsHelperTest extends TestCase
+{
+    public function testGetMostEfficientTypeData()
+    {
+        
+    }
+}
\ No newline at end of file
