diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 1f225bd..8dcd0a1 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -197,6 +197,10 @@ class ChurroTimeEntry
 
     public function setCookingAndCleanupDuration($cookingMinutes, $cleanupMinutes)
     {
+        if (!$this->getStartCookingAt()) {
+            throw new \Exception('Cannot set duration until startCookingAt is set!');
+        }
+
         $this->setEndCookingAt(
             (clone $this->getStartCookingAt())->modify(sprintf('+%s minutes', $cookingMinutes))
         );
