diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index abbaf42..9bb99eb 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -9,7 +9,6 @@ use Symfony\Component\Validator\Context\ExecutionContextInterface;
 /**
  * @ORM\Entity(repositoryClass="AppBundle\Repository\ChurroTimeEntryRepository")
  * @ORM\Table()
- * @Assert\Callback("validateDates")
  */
 class ChurroTimeEntry
 {
@@ -34,19 +33,16 @@ class ChurroTimeEntry
 
     /**
      * @ORM\Column(type="datetime")
-     * @Assert\NotBlank()
      */
     private $endCookingAt;
 
     /**
      * @ORM\Column(type="datetime")
-     * @Assert\NotBlank()
      */
     private $startCleanupAt;
 
     /**
      * @ORM\Column(type="datetime")
-     * @Assert\NotBlank()
      */
     private $endCleanupAt;
 
@@ -198,13 +194,4 @@ class ChurroTimeEntry
     {
         return ($this->getCleanupDuration() / 60) > 30;
     }
-
-    public function validateDates(ExecutionContextInterface $executionContext)
-    {
-        if ($this->getStartCookingAt() >= $this->getEndCookingAt()) {
-            $executionContext->buildViolation('Start must be before end')
-                ->atPath('startCookingAt')
-                ->addViolation();
-        }
-    }
 }
