diff --git a/src/AppBundle/Controller/ChurroTimeEntryController.php b/src/AppBundle/Controller/ChurroTimeEntryController.php
index 98dd80f..9007a96 100644
--- a/src/AppBundle/Controller/ChurroTimeEntryController.php
+++ b/src/AppBundle/Controller/ChurroTimeEntryController.php
@@ -51,9 +51,6 @@ class ChurroTimeEntryController extends Controller
     {
         $entry = new ChurroTimeEntry();
         $entry->setStartCookingAt(new \DateTime('now'));
-        $entry->setEndCookingAt(new \DateTime('now'));
-        $entry->setStartCleanupAt(new \DateTime('now'));
-        $entry->setEndCleanupAt(new \DateTime('now'));
         $form = $this->createForm(ChurroTimeEntryForm::class, $entry, [
             'csrf_protection' => false,
         ]);
