diff --git a/src/AppBundle/Form/ChurroTimeEntryForm.php b/src/AppBundle/Form/ChurroTimeEntryForm.php
index e0d264d..f49036d 100644
--- a/src/AppBundle/Form/ChurroTimeEntryForm.php
+++ b/src/AppBundle/Form/ChurroTimeEntryForm.php
@@ -31,6 +31,9 @@ class ChurroTimeEntryForm extends AbstractType
             ->add('startCookingAt', DateTimeType::class, [
                 'widget' => 'single_text'
             ])
+            ->add('cookingDuration', IntegerType::class, [
+                'label' => 'Cooking time (minutes)',
+            ])
         ;
     }
 }
