diff --git a/src/AppBundle/Form/ChurroTimeEntryForm.php b/src/AppBundle/Form/ChurroTimeEntryForm.php
index 6f50ed1..7c30cda 100644
--- a/src/AppBundle/Form/ChurroTimeEntryForm.php
+++ b/src/AppBundle/Form/ChurroTimeEntryForm.php
@@ -25,9 +25,6 @@ class ChurroTimeEntryForm extends AbstractType
             ->add('startCookingAt', DateTimeType::class, [
                 'widget' => 'single_text'
             ])
-            ->add('endCookingAt', DateTimeType::class)
-            ->add('startCleanupAt', DateTimeType::class)
-            ->add('endCleanupAt', DateTimeType::class)
             ->add('quantityMade', IntegerType::class)
             ->add('bakedBy', EntityType::class, [
                 'class' => Baker::class,
