diff --git a/src/AppBundle/Entity/ChurroTimeEntry.php b/src/AppBundle/Entity/ChurroTimeEntry.php
index 5725712..abbaf42 100644
--- a/src/AppBundle/Entity/ChurroTimeEntry.php
+++ b/src/AppBundle/Entity/ChurroTimeEntry.php
@@ -4,10 +4,12 @@ namespace AppBundle\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
 
 /**
  * @ORM\Entity(repositoryClass="AppBundle\Repository\ChurroTimeEntryRepository")
  * @ORM\Table()
+ * @Assert\Callback("validateDates")
  */
 class ChurroTimeEntry
 {
@@ -196,4 +198,13 @@ class ChurroTimeEntry
     {
         return ($this->getCleanupDuration() / 60) > 30;
     }
+
+    public function validateDates(ExecutionContextInterface $executionContext)
+    {
+        if ($this->getStartCookingAt() >= $this->getEndCookingAt()) {
+            $executionContext->buildViolation('Start must be before end')
+                ->atPath('startCookingAt')
+                ->addViolation();
+        }
+    }
 }
